packages.json

```diff
@@ -4,20 +4,16 @@
             "all": [
                 "alsa-firmware",
                 "android-udev-rules",
-                "apr",
-                "apr-util",
                 "distrobox",
                 "ffmpeg",
                 "ffmpeg-libs",
                 "ffmpegthumbnailer",
-                "fzf",
```

Comment:
We need this one for the just selectors in the terminal.

Reply:
I believe that generic bash dialog is used as a fallback if `fzf` & `gum` are not installed.
There is maybe some degradation in looks when using bash selector instead of `fzf` one, but it's still a viable solution I think.

So I also think that this one can be removed.

For final decision, I would consult @HikariKnight

Reply:
I'm personally indifferent on this package, but it was added over a year ago ( #84 ) to assist with `ujust` recipes. I am inclined to keep it without a strong reason to remove.